svgnative/include/svgnative/SVGRenderer.h

```diff
@@ -23,7 +23,7 @@ governing permissions and limitations under the License.
 #include <string>
 #include <tuple>
 #include <vector>
-
+#include "Rect.h"
```

Comment:
This should be after #include "Config.h"

Reply:
shifted above after config.h